pikaur/print_department.py

```diff
@@ -226,6 +227,14 @@ def pretty_format(pkg_update: 'InstallInfo') -> Tuple[str, SortKey]:  # pylint:d
         if pkg_update.devel_pkg_age_days:
             days_old = ' ' + _('({} days old)').format(pkg_update.devel_pkg_age_days)
 
+        if (
+                isinstance(pkg_update.package, AURPackageInfo) and
+                pkg_update.package.maintainer is None
+        ):
+            orphaned = ' [orphaned]'
```

Comment:
you need to use `_()` for all the string for the i18n translation

Reply:
(see 2 lines above as an example how to handle space correctly)

Reply:
so something like this?
```python3
orphaned = ' ' + _('[{}]').format("orphaned")
```

Reply:
```py
orphaned = f" [{_('orphaned')}]"
```

Reply:
updated it in d4a7759